Integration/LdapAuthIntegration.php

```diff
@@ -139,22 +151,32 @@ private function ldapUserLookup($ldap, $settings = [], $parameters = [])
         $query = $settings['user_query'];
         $is_ad = $settings['is_ad'];
         $ad_domain = $settings['ad_domain'];
+        $basic_auth= false;
 
-        $login = $parameters['login'];
+        if (isset($_SERVER["PHP_AUTH_USER"]) && !empty($_SERVER["PHP_AUTH_USER"])) {
+            $basic_auth= true;
+            $login = $_SERVER["PHP_AUTH_USER"];
+        }else
```

Comment:
Add missing space.
Add missing accolades.
Also format the code in the same way as the rest of the bundle.